app/src/main/res/layout/card_my_event_edit_request.xml

```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_weight="1">
+
+    <androidx.cardview.widget.CardView
+        android:id="@+id/id_event_edit_request_card"
```

Comment:
You can refactor out cardview and make it the root view

Reply:
Yes, it's done !